themes/bootstrap3/templates/RecordDriver/SolrDefault/core.phtml

```diff
@@ -94,6 +94,13 @@
 
       <? include(realpath(dirname(__FILE__) . "/JOP.phtml")); ?>
 
+      <? $issn = $this->driver->getCleanISSN(); if (!empty($issn)): ?>
+        <tr>
+          <th><?=$this->transEsc('Subito Delivery Service')?>: </th>
+          <td><a href="http://www.subito-doc.de/order/po.php?BI=KRIMDOK&VOL=&SS=<?=$issn?>">Link</a><br/></td>
```

Comment:
- The text 'Link' sould be translated. 
- 'Link' isn't a nice label for a link. Maybe something like 'Order now'?
- Is the br-tag needed here?